website/content/docs/install/production/requirements.mdx

```diff
@@ -189,9 +189,28 @@ This is not a supported or well-tested configuration. See [GH-13669][] for a
 further discussion and to provide feedback on your experiences trying to run
 rootless Nomad clients.
 
+## Running Nomad in Docker
```

Comment:
Should we throw a `~> Beta!` notice or something in here since it's so new and untested?

Not a blocker, just an idea.